setup.py

```diff
@@ -266,6 +267,28 @@
 import setuptools.command.sdist
 from setuptools import Extension, find_packages, setup
 from setuptools.dist import Distribution
+
+
+cwd = os.path.dirname(os.path.abspath(__file__))
```

Comment:
As we are in python-3.9+ world, would it be possible get use `pathlib` rather than `os.path`?

Reply:
Yes, we can. It should be done in a separate PR. Also, I want to note that `os.path` functions are slightly more efficient than `pathlib.Path` methods.